stripe/api_resources/invoice_item.py

```diff
@@ -6,4 +6,4 @@
 
 class InvoiceItem(CreateableAPIResource, UpdateableAPIResource,
                   ListableAPIResource, DeletableAPIResource):
-    OBJECT_NAME = 'invoice_item'
+    OBJECT_NAME = 'invoiceitem'
```

Comment:
We should probably cut a new version for this or something to prevent many future instances of confusion around this bad naming ...